pkg/capabilities/capabilities.go

```diff
@@ -174,6 +174,10 @@ type TargetCapability interface {
 	CallbackCapability
 }
 
+// DON represents a network of connected nodes.
+//
+// For an example of an empty DON check, see the following link:
+// https://github.com/smartcontractkit/chainlink/blob/develop/core/capabilities/transmission/local_target_capability.go#L31
```

Comment:
/nit the commit hash links age better :)

Reply:
I originally had that, but then ran into https://github.com/smartcontractkit/chainlink-common/pull/620#discussion_r1664885470 heh. I'd imagine it might be better if it breaks TBH. Then we'd know this needs to be updated anyway